src/Planet.cpp

```diff
@@ -139,8 +144,8 @@ void Planet::GetAtmosphericState(double dist, double *outPressure, double *outDe
 	// lapse rate http://en.wikipedia.org/wiki/Adiabatic_lapse_rate#Dry_adiabatic_lapse_rate
 	// the wet adiabatic rate can be used when cloud layers are incorporated
 	// fairly accurate in the troposphere
-	const double lapseRate_L = -m_surfaceGravity_g/specificHeatCp; // negative deg/m
-
+	// m_surfaceGravity_g is calculated and stored as a negative number
+	const double lapseRate_L = -m_surfaceGravity_g/specificHeatCp; // negative deg/m << wrong? -(-)/+ = +, should it be -abs(G)/hcp?
```

Comment:
Looks like github is warning that this line isn't indented correct? (we use tab, 4 char wide).